### DISCUSSION TITLE
Cannot start server in new window because no terminal app was specified

### DISCUSSION THREAD

**Discussion by Alexufo** [callstack/react-native-builder-bob#508] [Q&A]
1. npx create-react-native-library@latest awesome-library
2. Native - java&objc
3. yarn
4.yarn example android

I got `Cannot start server in new window because no terminal app was specified.`

**Comment by satya164** [comment]
I think you should open an issue in https://github.com/react-native-community/cli repo since the example app is just a regular React Native app.
